addons/gamification/static/src/css/gamification.css

```diff
@@ -14,8 +14,10 @@
   font-weight: bold;
   padding-top: 30px;
 }
-.openerp .oe_kanban_goal .oe_goal_state_block .oe_goal_state.oe_e {
-  font-size: 7em;
+.openerp .oe_kanban_goal .oe_goal_state_block .oe_goal_state .fa {
```

Comment:
maybe it's intended but you change the semantics: .class1.class2 and .class1 .class2